src/main.rs

```diff
@@ -19,11 +22,14 @@ use crate::subprocess::{Inheritance, SubProcess, SubProcessIntf};
 use crate::templates::Templates;
 use clap::ArgMatches;
 use color_eyre::eyre::Result;
+use prettytable::{Attr, Cell, Row, Table};
 use std::io::{self, Write};
 use std::process;
 use std::str::FromStr;
 use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
 
+const REDACTED: &str = "REDACTED";
```

Comment:
It might make sense for this to be the same value we show in the web UI: `"*****"`.

Reply:
Is that going to cause issues for scripting given the special handling of `*` in most languages? I don't mind changing it, but I don't want to cause additional pain for users.